support/Environments/frontera_gcc.sh

```diff
@@ -181,20 +182,23 @@ EOF
         tar xzf v6.10.2.tar.gz
         mv charm-6.10.2 charm
         cd $dep_dir/charm
-        ./build charm++ mpi-linux-x86_64 smp mpicxx --with-production -j6
-        ./build LIBS mpi-linux-x86_64 smp mpicxx --with-production -j6
+        MPICC=mpicc MPICXX=mpicxx \
+             ./build charm++ mpi-linux-x86_64 smp mpicxx --with-production -j6
+        MPICC=mpicc MPICXX=mpicxx \
+             ./build LIBS mpi-linux-x86_64 smp mpicxx --with-production -j6
```

Comment:
I'm getting a few build errors that seem to get ignored, is that expected? Example:

```
gfortran: error: unrecognized command line option ‘-auto’
Fatal Error by charmc in directory /scratch1/06739/nfischer/spectre/spectre_deps/charm/mpi-linux-x86_64-smp-mpicxx/tmp/libs/ck-libs/ampi
   Command mpif90 -auto -fPIC -I/scratch1/06739/nfischer/spectre/spectre_deps/charm/mpi-linux-x86_64-smp-mpicxx/bin/../include -O2 -c ampifimpl.f90 -o /scratch1/06739/nfischer/spectre/spectre_deps/charm/mpi-linux-x86_64-smp-mpicxx/lib/ampifimpl.o returned error code 1
charmc exiting...
gmake[3]: [/scratch1/06739/nfischer/spectre/spectre_deps/charm/mpi-linux-x86_64-smp-mpicxx/lib/libmoduleampif.a] Error 1 (ignored)
```

Another example:

```
checking dynamic linker characteristics... g++: error: moduleinit275155.o: No such file or directory
ar: creating ../../../../lib/libmodulepose.a
/bin/cp libmodulepose.dep ../../../../lib/.
gmake[3]: Leaving directory `/scratch1/06739/nfischer/spectre/spectre_deps/charm/mpi-linux-x86_64-smp-mpicxx/tmp/libs/ck-libs/pose'
```

Reply:
Hard to be sure with charm.  The ampifimpl.o build is marked ok-to-fail in the makefile, and I know there are other places where they try to build multiple things and just use whichever works, so might be intentional.  The "-auto" part seems odd in that case, though.  Might be assuming the mpif* compilers are Intel.

The second error is harder to pin down.

Neither seems to cause us problems, in any case.